changelog.txt

```diff
@@ -1,3 +1,38 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Add: Make mNewGRF Scanner / World Generation update more smooth and make aborting them react faster (#8830)
```

Comment:
Why do you have `mNewGRF Scanner` instead of just `NewGRF Scanner`?

Reply:
Because people make typos? No need to be mean about it :P